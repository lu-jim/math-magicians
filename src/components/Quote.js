import React from 'react';

const Quote = () => (
  <div id="quote">
    <p style={{ fontStyle: 'italic' }}>
      Mathematics as an expression of the human mind reflects the active will,
      the contemplative reason, and the desire for aesthetic perfection.
      Its basic elements are logic and intuition, analysis and construction,
      generality and individuality.
      â€” Richard Courant, German-American mathematician
    </p>
  </div>
);

export default Quote;
