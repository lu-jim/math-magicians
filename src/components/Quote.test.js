import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('renders home correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
