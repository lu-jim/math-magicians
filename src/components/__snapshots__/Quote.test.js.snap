// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home correctly 1`] = `
<div
  id="quote"
>
  <p
    style={
      Object {
        "fontStyle": "italic",
      }
    }
  >
    Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. â€” Richard Courant, German-American mathematician
  </p>
</div>
`;
